src/ucp/core/ucp_ep.h

```diff
@@ -511,8 +543,7 @@ typedef struct ucp_conn_request {
     uct_device_addr_t           *remote_dev_addr;
     struct sockaddr_storage     client_address;
     ucp_ep_h                    ep; /* valid only if request is handled internally */
-    ucp_wireup_sockaddr_data_t  sa_data;
-    /* packed worker address follows */
+    /* sa_data and packed worker address follow */
```

Comment:
maybe add `ucp_wireup_sockaddr_data_base_t sa_data[0];`?

Reply:
imo it will not make it  more clear, such approach is typically used for inlined variable size array

Reply:
to simplify usage, for examle here https://github.com/openucx/ucx/pull/7457#discussion_r718380139

Reply:
still does not look better to me, @yosefe wdyt?